core/templates/dev/head/pages/exploration_player/SupplementalCardDirective.js

```diff
@@ -0,0 +1,88 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controller for the supplemental card.
+ */
+
+oppia.directive('supplementalCard', [function() {
+  return {
+    restrict: 'E',
+    scope: {
+      showNextCard: '&',
+      onSubmitAnswer: '&'
+    },
+    templateUrl: 'components/SupplementalCard',
+    controller: [
+    '$scope', '$window', 'oppiaPlayerService', 'UrlInterpolationService',
+    'playerPositionService', 'playerTranscriptService',
+    'ExplorationObjectFactory', 'windowDimensionsService',
+    function($scope, $window, oppiaPlayerService, UrlInterpolationService,
+      playerPositionService, playerTranscriptService, explorationObjectFactory,
```

Comment:
Careful -- dependency var should match case of its stringly-typed counterpart (for the exploration object factory dependency). Also, suggest moving it to next line so it's easy to compare the two lists.